src/System.Private.CoreLib/src/Internal/Runtime/Augments/RuntimeThread.Unix.cs

```diff
@@ -20,21 +23,33 @@ public sealed partial class RuntimeThread
 
         internal WaitSubsystem.ThreadWaitInfo WaitInfo => _waitInfo;
 
+#if MONO
+        private void Unix_PlatformSpecificInitialize()
```

Comment:
I saw Ludovic made initial commits to change builds profiles [here](https://github.com/mono/mono/commits/gsoc-System.IO.Pipes). But I at the moment don't know how I could refactor corlib dll.sources to use these profiles (win32, linux, unix).  I need to investigate this more deeply.